src/YoffeeElement.js

```diff
@@ -49,7 +49,7 @@ class YoffeeElement extends HTMLElement {
         // This is here for when users call `super.connectedCallback()`. Prevent the crash :)
     }
 
-    propUpdatedCallback() {
+    propUpdatedCallback(prop) {
```

Comment:
is this change intended?

Reply:
Yes, it received the prop name and now typescript needs to know it.